### PR TITLE
Add eCAPRIS field to "Funding" tab and adjust required fields #8079

### PR DESCRIPTION
**Pull request by sergiogcx**
## Associated issues

Closes https://github.com/cityofaustin/atd-data-tech/issues/8079

## Testing

https://deploy-preview-523--atd-moped-main.netlify.app/moped/projects/345?tab=funding

**Steps to test:**

https://user-images.githubusercontent.com/5282430/149398478-a73d0a51-4dfa-4766-b087-cdb71afeddbf.mp4

https://user-images.githubusercontent.com/5282430/149398512-81ac9caf-351e-4dd5-a011-c82cb1a41f28.mp4

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/ProjectFundingTable.js [hunk 1]
```diff
@@ -324,96 +382,84 @@ const ProjectFundingTable = () => {
...
-                      </span>
-                    ),
-                    severity: "error",
-                  });
+            return updateProjectFunding({
+              variables: updateProjectFundingData,
+            })
+              .then(() => refetch())
+              .catch(error => {
+                setSnackbarState({
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/projectView/ProjectFundingTable.js:432]: Thanks for including this pattern, its really nice.

File: moped-editor/src/views/projects/projectView/ProjectFundingTable.js [hunk 2]
```diff
@@ -324,96 +382,84 @@ const ProjectFundingTable = () => {
...
+            updateProjectFundingData.funding_amount = Number(
+              newData.funding_amount
+            );
+            // add fallback of empty strings instead of null value
+            updateProjectFundingData.fund_dept_unit =
+              newData.fund_dept_unit || "";
+            updateProjectFundingData.funding_description =
+              newData.funding_description || "";
+            updateProjectFundingData.funding_program_id =
+              newData.funding_program_id || 0;
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/projectView/ProjectFundingTable.js:425]: It's tedious that we have to do all this data cleaning but I don't know of a better solution. Just feels like a lot of extra boilerplate.

**Review by mateoclarke — APPROVED** [review]
🚢 💨 ✨

**Review by amenity — CHANGES_REQUESTED** [review]
Looks great, @sergiogcx ! Will you have the eCAPRIS field display "None" if it's blank like it does on the "Summary" tab?

**Comment by sergiogcx** [comment]
@amenity ready for review again

**Comment by amenity** [comment]
@sergiogcx - so sorry but I see that the mock up we had in the issue was inconsistent with our other designs. Can you please style the "eCAPRIS Subproject ID" text heading to match [that on the "Summary" tab](https://user-images.githubusercontent.com/1463708/149555083-28d54012-8b74-440d-8238-14e5a4338118.png) and remove the colon?

Funding-tab-eCapris-task-order

**Comment by sergiogcx** [comment]
@amenity can you please check again?

**Review by amenity — APPROVED** [review]
👌🙏🚢

**Pull request merged**